about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/styles/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:wght@300&display=swap"
+    rel="stylesheet">
+  <title>Mike's Capstone Project</title>
+</head>
+<body>
+  <!-- the navbar (and header) section -->
+  <nav class="mini-nav on-desktop">
+    <ul class="mininav-menu">
+      <li class="mininav-item"><img src="/img/facebook.svg" alt="facebook"></li>
+      <li class="mininav-item"><img src="/img/twitter.svg" alt="twitter"></li>
```

Comment:
- [ ] nice job. Please ensure your work is responsive from mobile to the desktop version
![Screenshot (1082)](https://github.com/Mike111222/CapstoneProject/assets/65298048/2a847667-6c67-4cce-812c-eb48314b63c1)

Reply:
Thanks a lot for you kind feedback sir. I have implemented the changes suggested with responsive units.